src/fastoad/models/performances/mission/openmdao/mission.py

```diff
@@ -408,6 +475,27 @@ def as_scalar(value):
         if self.options["out_file"]:
             self.flight_points.to_csv(self.options["out_file"])
 
+    def _compute_taxi_out(self, inputs, outputs, propulsion_model):
+        """
+        Computes the taxi-out segment.
+        """
+        start_of_taxi_out = FlightPoint(
+            altitude=inputs[self._mission_vars.TAKEOFF_ALTITUDE.value],
+            true_airspeed=0.0,
+            # start mass is irrelevant here as long it does not get negative during computation.
```

Comment:
Can't we compute the ramp weight on the same model as TOW computation through '_get_tow_component" to have a relevant weight here even if it would not change the results ? The ramp weight could then be added to the variable list which could be an interesting information.